pkg/autoupdate/findnew.go

```diff
@@ -317,10 +317,10 @@ func getPossibleUpdates(ctx context.Context, updateDir, binaryName string) ([]st
 	return possibleUpdates, nil
 }
 
-// FindBaseDir takes a binary path, that may or may not include the
+// findBaseDir takes a binary path, that may or may not include the
```

Comment:
Do we still need this, and anything else in the FindNewest?

Reply:
Yeah, we fall back to the legacy autoupdate library currently if there's nothing in the new library. We can definitely get rid of that at some point, but I wasn't in a rush to rip that part out as quickly.